contracts/libraries/CappedMath.sol

```diff
@@ -25,10 +25,10 @@ library CappedMath {
     }
 
     /**
-     * @dev Subtracts two integers, returns 0 on underflow.
+     * @dev Subtracts two integers, returns 0 on underflow or equal values.
      */
     function subCap(uint _a, uint _b) internal pure returns (uint) {
-        if (_b > _a)
+        if (_b >= _a)
```

Comment:
">" was OK as if they are equal l34 will return 0.
_a==_b is not an edge case so better treated normally. And "a>=b" is more gas intensive than "a>b" as from a compiler perspective it expends to "a>b || a==b".

Reply:
Understood.